### ISSUE TITLE
Fix sign-out action

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/ember-auth#6]
Broke when switching from a sign-out route to a simple link.

Turns out, I should have just deleted the template and kept the route.

**Comment by faetea** [comment]
I changed `application/template.hbs` to `{{my-application signOut="signOut"}}`, 
and added `auth: Ember.inject.service(),` in `app/application/route.js`

Looks good to me:

``` ruby
Started DELETE "/sign-out/1" for 127.0.0.1 at 2016-04-06 17:09:31 -0400
Processing by UsersController#signout as JSON
  Parameters: {"id"=>"1"}
  User Load (0.3ms)  SELECT  "users".* FROM "users" WHERE "users"."token" = $1 LIMIT 1  [["token", "c0b17872b5b6b5b228ecfa366de38619"]]
  User Load (0.3ms)  SELECT  "users".* FROM "users" WHERE "users"."id" = $1 LIMIT 1  [["id", 1]]
  SQL (1.4ms)  UPDATE "users" SET "token" = 'fe2a50bfeea0d0aebace0db7dee352c6', "updated_at" = '2016-04-06 21:09:31.309093' WHERE "users"."id" = $1  [["id", 1]]
Completed 204 No Content in 5ms (ActiveRecord: 2.0ms)
```

**Comment by gaand** [comment]
Closed in 3fcde7576d45ca076be49a1a9008e8a09d9f604a

Merge of @faetea's pull request.
